src/js/disability-benefits/components/ClaimPhase.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Link } from 'react-router';
+import moment from 'moment';
+import _ from 'lodash/fp';
+import { getUserPhaseDescription } from '../utils/helpers';
+
+const stepClasses = {
+  1: 'one',
+  2: 'two',
+  3: 'three',
+  4: 'four',
+  5: 'five last'
+};
+
+const COMPLETE_PHASE = 5;
+const INITIAL_ACTIVITY_ROWS = 5;
+
+function getClasses(phase, current) {
+  const processClass = 'step wow fadeIn animated';
+  const stepClass = stepClasses[phase];
+  if (phase === current) {
+    return `${stepClass} ${processClass} section-current`;
+  } else if (current > phase) {
+    return `${stepClass} ${processClass} section-complete`;
+  }
+
+  return `${stepClass} ${processClass}`;
+}
+
+
+export default class ClaimPhase extends React.Component {
+  constructor(props) {
+    super();
+    this.state = { open: props.current === props.phase, showAll: false };
+    this.expandCollapse = this.expandCollapse.bind(this);
+    this.displayActivity = this.displayActivity.bind(this);
+    this.showAllActivity = this.showAllActivity.bind(this);
+  }
+  displayActivity() {
+    if (this.props.activity) {
+      const filesPath = `your-claims/${this.props.id}/files`;
+      const limitList = this.state.showAll ? _.identity : _.take(INITIAL_ACTIVITY_ROWS);
+      const activityList = limitList(this.props.activity).map((activity, index) =>
+        <div key={index} className="claims-evidence">
+          <p className="claims-evidence-date">{moment(activity.date).format('MMM D, YYYY')}</p>
+          {activity.type === 'still_need_from_you_list'
```

Comment:
@austin-martinez Does this make sense as far as using the different event types?